alpaca/version.go

```diff
@@ -0,0 +1,38 @@
+package alpaca
+
+import (
+	"runtime/debug"
+	"strings"
+	"sync"
+)
+
+const repoName = "github.com/alpacahq/alpaca-trade-api-go"
+
+var (
+	once            = sync.Once{}
+	encodedVersions string
+)
+
+// GetVersion returns running go version and alpaca-trade-api-go version
+func GetVersion() string {
+	once.Do(func() {
```

Comment:
minor: `init` is evil, but instead of `sync.Once` we can do it as https://github.com/uber-go/guide/blob/master/style.md#avoid-init suggests:
```go
const repo = "github.com/alpacahq/alpaca-trade-api-go"

var version = initVersion()

func initVersion() string {
    moduleVersion := "unknown"
    if buildInfo, ok := debug.ReadBuildInfo(); ok {
		for _, dep := range buildInfo.Deps {
			if strings.HasPrefix(dep.Path, repo) {
				moduleVersion = dep.Version
				break
			}
		}
    }
    return fmt.Sprintf("ApcaGo/%s GoRuntime/%s", moduleVersion, runtime.Version())
}

func Version() string {
    return version
}
```

Reply:
Nice Catch! 🙏🏼 

Addressed: https://github.com/alpacahq/alpaca-trade-api-go/pull/254/commits/689e33c0262c4cc4e0f01fe704c8a2a8ea9248a8